azure-pipelines.yml

```diff
@@ -253,6 +253,9 @@ stages:
               artifactType: Container
               parallel: true
 
+        - powershell: eng\SetupVSHive.ps1
```

Comment:
We use this now to set the OOP64 for Roslyn to "off" (since they won't get our binding-redirects.